app/assets/javascripts/views/component/test-executor.coffee

```diff
@@ -31,6 +31,8 @@ class Crucible.TestExecutor
     @element.find('.selectDeselectAll').click(@selectDeselectAll)
     @element.find('.expandCollapseAll').click(@expandCollapseAll)
     @element.find('.filter-by-executed a').click(@showAllSuites)
+    $('.edit-server-name-icon').click(@toggleEditDialogue)
+    $('.submit-server-name').click(@editServerName)
```

Comment:
this should not go into test executor... it has nothing to do with executing tests.